src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format(ttf),
+    url(./fonts/Avenir-Heavy.ttf) format(ttf);
+
+}
+
+.form-search1 {
+  margin: 14px 0;
+  height: 68px;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: medium;
+  background-color: #fff;
+  background-image: url(Search.svg);
+  background-position: 25px 23px;
+  background-repeat: no-repeat;
+
+  padding-left: 60px;
+}
+
+.form-search2 {
+  margin: 0;
+  height: 37px;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: small;
+  background-color: #fff;
+  background-image: url(Search.svg);
+  background-position: 13px;
+  background-repeat: no-repeat;
+  background-size: 1%;
+  padding-left: 35px;
+}
```

Comment:
Same. Follow the checklist
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/99403198/226096082-f33a91d3-45ec-437b-a00f-8a956e7ef5bf.png">